modules/osd-aws-privatelink-about.adoc

```diff
@@ -2,6 +2,6 @@
 [id="osd-aws-privatelink-about.adoc_{context}"]
 = Understanding AWS PrivateLink
 
-A {product-title} cluster can be created without any requirements on public subnets, internet gateways, or network address translation (NAT) gateways. In this configuration, Red Hat uses AWS PrivateLink to manage and monitor a cluster in order to avoid all public ingress network traffic.
+A {product-title} cluster can be created without any requirements on public subnets, internet gateways, or network address translation (NAT) gateways. In this configuration, Red Hat uses AWS PrivateLink to manage and monitor a cluster in order to avoid all public ingress network traffic. Because of this configuration, AWS PrivateLink clusters do not support the Ingress Operator.
```

Comment:
This is not entirely accurate. AWS PrivateLink clusters do indeed have an Ingress Operator, but it's just not on a public subnet. It creates a load balancer on their private subnet.